plugin/builtin/prefixtransformer/PrefixTransformer.go

```diff
@@ -7,26 +7,23 @@ package main
 import (
 	"errors"
 
-	"sigs.k8s.io/kustomize/api/filters/prefixsuffix"
+	"sigs.k8s.io/kustomize/api/filters/prefix"
 	"sigs.k8s.io/kustomize/api/resmap"
 	"sigs.k8s.io/kustomize/api/types"
 	"sigs.k8s.io/kustomize/kyaml/resid"
-	"sigs.k8s.io/yaml"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
-// Add the given prefix and suffix to the field.
+// Add the given prefix to the field
 type plugin struct {
 	Prefix     string        `json:"prefix,omitempty" yaml:"prefix,omitempty"`
-	Suffix     string        `json:"suffix,omitempty" yaml:"suffix,omitempty"`
 	FieldSpecs types.FsSlice `json:"fieldSpecs,omitempty" yaml:"fieldSpecs,omitempty"`
 }
 
 //noinspection GoUnusedGlobalVariable
 var KustomizePlugin plugin
 
-// A Gvk skip list for prefix/suffix modification.
-// hard coded for now - eventually should be part of config.
```

Comment:
My comment might have become hidden in the large pile of other comments but please don't forget, I think it would be best to keep this comment in, or even change it to a TODO such as 

```
TODO: Make this gvk skip list part of the config.
```

both here and in the SuffixTransformer.

Reply:
Added the TODO 👍🏼